RallyHereIntegration/Source/RallyHereIntegration/Private/RH_SessionData.cpp

```diff
@@ -306,22 +320,26 @@ void URH_SessionView::ForcePollForUpdate(bool bClearEtag)
 
 void URH_SessionView::AddDeferredPoll(const FRH_DeferredSessionPoll& DeferredPoll)
 {
+	// only store latest notification poll
 	if (DeferredPoll.PollType == FRH_DeferredSessionPoll::Type::Notification)
 	{
-		// remove any existing notification polls, as this one supercedes them
+		// strip any existing notification polls we can, as this one supercedes them
 		for (int i = DeferredPolls.Num() - 1; i >= 0; --i)
 		{
-			if (DeferredPolls[i].PollType == FRH_DeferredSessionPoll::Type::Notification)
```

Comment:
lol oh, this just always removed all deferred polls except for the one being added?  That's fascinating lol

Reply:
Yeah, notifications are ordered, so only the newest one matters (assuming nothing is handling callbacks).  However, notifications for previous versions could still cause us to think there was an etag change when we are already up to date (since etags are not ordered or comparable on the client).  So this just uses notification processing order as a way to strip out the older notifications.